physionet-django/user/validators.py

```diff
@@ -255,3 +256,11 @@ def validate_domain_list(value):
     """
     if not re.fullmatch(r'(\w+\.\w+,*\s*)*', value):
         raise ValidationError('Must be separated with commas.')
+
+
+def validate_training_file_size(value):
+    """
+    Validate the file size of a file.
+    """
+    if value.size > settings.MAX_TRAINING_REPORT_UPLOAD_SIZE_MB * 1024 * 1024:
```

Comment:
HI, This seems some kind of mistake, i remember removing the MB stuff and doing bytes as Benjamin suggested. 

it might happened because i was poking around multiple branches and lost my latest changes. Pushing the changes with the fix asap